sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3276,7 +3276,7 @@ SUBROUTINE MISCLN
            DPBND = 300.E2
            CALL CALCAPE(ITYPE,DPBND,P1D,T1D,Q1D,LB2,EGRID1,     &
                         EGRID2,EGRID3,EGRID4,EGRID5)
-           IF (SUBMODELNAME == 'RTMA') MUMIXR(I,J) = Q1D(I,J)
+!           IF (SUBMODELNAME == 'RTMA') MUMIXR(I,J) = Q1D(I,J)
```

Comment:
Why to comment out this line? If this is not needed by RTMA, we should remove this line.